packages/core/src/components/Tabs/Tab.jsx

```diff
@@ -33,12 +33,13 @@ export class Tab extends React.PureComponent {
     return (
       <a
         aria-selected={String(this.props.selected)}
+        aria-disabled={String(this.props.disabled)}
         aria-controls={this.props.panelId}
         className={classes}
         href={this.href}
         id={this.props.id}
-        onClick={this.handleClick}
-        onKeyDown={this.handleKeyDown}
+        onClick={!this.props.disabled && this.handleClick}
+        onKeyDown={!this.props.disabled && this.handleKeyDown}
```

Comment:
Could we add a unit test to confirm these events aren't fired when `disabled` is `true`? Check out the `'calls onClick'` test in `Tab.test.jsx` for an example.